### ISSUE TITLE
CI/CD to ensure endo, agoric-sdk references in docs site is up-to-date and error-free

### ISSUE THREAD

**Issue by LuqiPan** [Agoric/documentation#1046]
With the introduction of #1038 and as we work towards #927, we enter a world where the build and publication of docs.agoric.com depends on both Agoric/agoric-sdk and endojs/endo repos.

Once #1038 is complete, we'll have Agoric/documentation pulling from and building reference docs for both Agoric/agoric-sdk and endojs/endo repos whenever PRs are open or merged into Agoric/documentation repo.

However, docs site won't update when changes are made to Agoric/agoric-sdk and endojs/endo repos. In order to achieve that, we'd need to complete these tasks below.

| Trigger    | Repo              | Goal                                 | Issue |
|------------|-------------------|--------------------------------------|-------|
| Open a PR  | Agoric/agoric-sdk | Verify docs site still builds        |    #1044   |
| Merge a PR | Agoric/agoric-sdk | Deploy to docs site with new changes |   #1042     |
| Open a PR  | endojs/endo       | Verify docs site still builds        |   #1045    |
| Merge a PR  | endojs/endo       | Deploy to docs site with new changes |   #1043    |
